.github/workflows/build.yml

```
@@ -23,68 +24,50 @@ jobs:
-            USERNAME=paf
-            USER_UID=1000
-            USER_GID=1000
+            PAF_USERNAME: paf
+            PAF_UID: 1000
+            PAF_GID: 1000
+            BASE_FLAVOUR: cuda
 
       - name: Build and push Docker image
         id: build
         if: github.event_name != 'pull_request'
         uses: docker/build-push-action@v3
```

actionlint (1.7.8)

the runner of "docker/build-push-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)